collectors/aws/collector.js

```diff
@@ -812,6 +816,13 @@ var postcalls = [
                 filterValue: 'FunctionArn'
             }
         },
+        RDS: {
```

Comment:
does this need to be a custom collector call?

Reply:
No. I had confusion earlier but you clerified in the meeting and I have removed the custom collector for this postcall.